setup.py

```diff
@@ -30,7 +30,7 @@
     ],
     entry_points='''
         [console_scripts]
-        clickme=vumi_http_proxy.clickme:cli
+        queen_of_ni=vumi_http_proxy.queen_of_ni:cli
```

Comment:
We could also name the script `queen-of-ni` (just the LHS, not the Python module)? Not sure whether you like `-` or `_` more in command names?